api/app/tests/apps/api/v1/test_endpoints.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from signals.apps.signals.models import Signal
+from tests.apps.signals.factories import NoteFactory, SignalFactory
+from tests.test import SIAReadWriteUserMixin, SignalsBaseApiTestCase
+
+THIS_DIR = os.path.dirname(__file__)
+SIGNALS_TEST_DIR = os.path.join(os.path.split(THIS_DIR)[0], '..', 'signals')
+
+
+class TestPrivateEndpoints(SIAReadWriteUserMixin, SignalsBaseApiTestCase):
+    """Test whether the endpoints in V1 API """
```

Comment:
we need to fix or remove this docstring, as-is it is useless

Reply:
Done